trav/main.py

```diff
@@ -86,14 +108,24 @@ def __getattr__(self, variable):
 
 
 class Travis(metaclass=TravisVariables):
-    @staticmethod
-    def get_svg_from_badge_url(url):
+    TRAVIS_BASE_URL = 'https://api.travis-ci.org/'
+    TRAVIS_BADGE_URL = TRAVIS_BASE_URL + '{group}/{repo}.svg?branch={branch}'
+
+    @classmethod
+    def get_svg_from_badge_url(cls, url):
```

Comment:
this doesnt use `cls`, so why is it a `classmethod`?

Reply:
My bad. Changed back to `staticmethod`